setup.py

```diff
@@ -24,13 +24,13 @@
       license='MIT License',
       packages=find_packages(),
 
-      python_requires='>=3.9',
+      python_requires='<4',

```

Comment:
Here I am really leanant with required python version.You could also decide to only allow for the new version ">=3.9,<=3.12.2" but you would have the same issue for each newer python realeases...